pkg/sql/backfill/backfill.go

```diff
@@ -477,6 +477,10 @@ type VectorIndexHelper struct {
 	vecIndex *cspann.Index
 	// indexPrefix are the prefix bytes for this index (/Tenant/Table/Index).
 	indexPrefix []byte
+	// tableDesc is the table descriptor for the table being backfilled.
+	tableDesc catalog.TableDescriptor
+	// indexDesc is the index descriptor for the index being backfilled.
```

Comment:
Where is `indexDesc` being initialized and used? I see where `tableDesc` is being used, but not `indexDesc`.